certs/bootstrap/certs.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package bootstrap
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/mainflux/mainflux/certs"
+	"github.com/mainflux/mainflux/pkg/errors"
+
+	"os"
+	"sync"
+
+	mfsdk "github.com/mainflux/mainflux/pkg/sdk/go"
+)
+
+const authHeaderPrefix = "Bearer"
+
+var (
+	// ErrUnauthorizedAccess indicates missing or invalid credentials provided
+	// when accessing a protected resource.
+	ErrUnauthorizedAccess = errors.New("unauthorized access to Bootstrap service")
+
+	// ErrUnexpectedBSResponse indicates unexpected response from Bootstrap service.
+	ErrUnexpectedBSResponse = errors.New("unexpected Bootstrap service response")
+
+	ErrUnableToAccess = errors.New("unable to access bootstrap service")
```

Comment:
Add comments for exported errors.

Reply:
Please don't forget to address this one.

Reply:
done